files/ko/web/javascript/reference/global_objects/dataview/setint8/index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: DataView.prototype.setInt8()
+slug: Web/JavaScript/Reference/Global_Objects/DataView/setInt8
+l10n:
+  sourceCommit: e01fd6206ce2fad2fe09a485bb2d3ceda53a62de
+---
+
+{{JSRef}}
+
+{{jsxref("DataView")}} 인스턴스의 **`setInt8()`** 메서드는 숫자를 받아 이 `DataView`의 지정된 바이트 오프셋에 있는 바이트에 8비트 부호 있는 정수로 저장합니다.
+
+{{EmbedInteractiveExample("pages/js/dataview-setint8.html")}}
+
+## 구문
+
+```js-nolint
+setInt8(byteOffset, value)
+```
+
+### 매개변수
+
+- `byteOffset`
+  - : 데이터를 저장할 데이터 뷰의 시작점으로부터의 바이트 단위의 오프셋.
+- `value`
+  - : 저장할 값. 어떻게 값이 바이트로 인코딩되는지는 [값 인코딩 및 정규화](/ko/docs/Web/JavaScript/Reference/Global_Objects/TypedArray#value_encoding_and_normalization)를 보시기 바랍니다.
+
+### 반환 값
+
+{{jsxref("undefined")}}.
+
+### 예외
+
+- {{jsxref("RangeError")}}
+  - : 뷰의 끝을 넘어 읽는 수도 있는 `byteOffset`이 설정된 경우 발생합니다.
```

Comment:
> Thrown if the byteOffset is set such that it would store beyond the end of the view.




```suggestion
  - : 뷰의 끝을 넘어서 저장하려고 하는 경우에 byteOffset이 설정되면 발생합니다
```

저장이라는 단어가 들어가면 좋을 것 같은데 위 정도로 변경하면 어떨까요??

Reply:
하핫 네 복붙의 폐해였습니다 😂